### DISCUSSION TITLE
spd-say  speech-dispatcher integration?

### DISCUSSION THREAD

**Discussion by grigio** [coqui-ai/TTS#2219] [General]
is possible to use with it ? Many programs use that wrapper

https://manpages.debian.org/testing/speech-dispatcher/spd-say.1.en.html

**Comment by nose-gnome** [comment]
I've been looking everywhere to see if it is possible to integrate it with spd-say, but I haven't found anything so far

**Comment by PeteHemery** [comment]
Here's my method
https://github.com/coqui-ai/TTS/discussions/2583#discussioncomment-8803059
